mage_ai/streaming/sources/rabbitmq.py

```diff
@@ -0,0 +1,98 @@
+from mage_ai.shared.config import BaseConfig
+from mage_ai.streaming.sources.base import BaseSource
+from dataclasses import dataclass
+from typing import Callable, Dict
+from collections import namedtuple
+import pika
+from pika.exceptions import AMQPConnectionError
+
+@dataclass
+class ConsumeConfig:
+    auto_ack: bool = False
+    exclusive: bool = False
+    inactivity_timeout: float = None
+
```

Comment:
nit: remove extra empty lines. keep two lines between classes. could you use flake8 linter to check the code style?

Reply:
Fixed on new commit (57b943b2dfdcf4d1ab128bf663facbd64e20beee)